lib/pf/Switch.pm

```diff
@@ -2726,40 +2727,60 @@ Default implementation.
 =cut
 
 sub returnRadiusAccessAccept {
-    my ($self, $vlan, $mac, $port, $connection_type, $user_name, $ssid, $wasInline, $user_role) = @_;
+    my ($self, $args) = @_;
     my $logger = $self->logger();
 
-    # Inline Vs. VLAN enforcement
+    my $filter = pf::access_filter::radius->new;
+    my $rule = $filter->test('returnRadiusAccessAccept', $args);
     my $radius_reply_ref = {};
+    $radius_reply_ref = $filter->handleAnswerInRule($rule,$args);
```

Comment:
Yes it should be possible to have a attribut in radius_filters.conf to tell to merge all the attributes or no.
But let's say we use the 2960 module if i call the radius filter at the end it mean that first i have to call @{$self->SUPER::returnRadiusAccessAccept($args)}; but it mean that i will call the radius filter from returnRadiusAccessAccept's Switch.pm and at the end of returnRadiusAccessAccept's 2960 so twice.
So i have to find a work arround ...